chain/jsonrpc/src/lib.rs

```diff
@@ -288,12 +288,13 @@ struct JsonRpcHandler {
 }
 
 impl JsonRpcHandler {
-    pub async fn process(&self, message: Message) -> Result<Message, HttpError> {
+    pub async fn process(&self, message: Message) -> Result<Message, RpcError> {
```

Comment:
Sorry, I'm not super familiar with the codebase in this area, but when I was taking a look at the `Message` enum, it does seem to have a function to convert RpcError to a response. Is that what we might want to use here, maybe just return the `Message` with RpcError instead of `Result<Message, RpcError>`?

Or maybe better, since we know we always expect a "response" out of this function, why not return an object of type `Response` defined [here](https://github.com/near/nearcore/blob/fe4ac1a316cf09fa391636d8532192239f8b5a1c/chain/jsonrpc-primitives/src/message.rs#L76)?

Reply:
Additionally, I see this function has only one call site which is in the same file, might be good to get rid of the `pub` here.